Neptune outfit submitions

Neptune-Zvezda opened the pull request:
## About the PR
I have added 18 New Pairs of Striped Socks using the coder socks as the base
(as well as Slightly editing the old Coder Socks with 4 pixels at the top)
and added these to the Pride Vendor, in addion I have added a personal winter coat Labled as N-Parka which used the "coat" in winter coats as the base. 

## Why / Balance
There was only two pairs of  socks and when asking others, agreed that more would be appricated

## Technical details
The Pride Vendor inventory as been edited to currently hold all basic white and _____  socks while department themed have been moved to their respective "drobe"s for acquisition. 
## Media
Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
--> 
![Proof it fucking works](https://github.com/DeltaV-Station/Delta-v/assets/154393072/fa90b3f2-95f8-4635-88c4-dc86e41e965e)
![ProofHeadlockers1](https://github.com/DeltaV-Station/Delta-v/assets/154393072/82c00481-e774-462f-97ee-7009131bef6d)
![ProofHeadlockers2](https://github.com/DeltaV-Station/Delta-v/assets/154393072/7a35834a-3573-43a4-a192-f560772ed766)
![ProofPrideVendor](https://github.com/DeltaV-Station/Delta-v/assets/154393072/d2f51189-6cb3-4d80-b900-9fbf8259f39e)
![ProofSpawnMenu](https://github.com/DeltaV-Station/Delta-v/assets/154393072/5f926056-8e11-4efe-a18b-fc8a80136987)
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->
Reminder to Self and others to do not capitalize the equip-FEET.png (or other part) and icon.png files as its hard to update them to work
(will post in comments full log for access as i do not know how to link a download to the txt itself)
**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
--> 
Current balance changes include a adjustment to the price of the pride vendor restock to 1300 as the addions in it increase its value, this is to avoid being able to make profit by reselling bought restock crates
Current version does not have "captain" socks within the Locker as the Limit of 30 items was reached, Potential solutions include a captain drobe to move most extra clothing out of locker to clear for needed items, Alternitively adding all "socks" too their own custom Sock Vending Machine and custom Cargo Crates may work best as they are purely vanity.
Plans to Add Some Functional Sock variants Like No slips for Janitor and perhaps a storage for Sec designs for a knife
<!--

Neptune-Zvezda committed:
All But Yml's and Parka made

This is suffering

Neptune-Zvezda committed:
Sock Update .2

Should be all working but fact check it

IamVelcroboy commented:
We need a feature, if you're wearing socks, you can slide on wood floors.

leonardo-dabepis commented:
I like the idea of this, and most of the spritework is good. I have a couple of nitpicks though.

- Some of the icon sprites aren't centered properly. These would be Captain_Blue_and_Gold, Captain_Blue_and_White, Captain_Gold_and_Blue, Cargo, Cargo_LO-QM, Atmos, Engineering, Science, Security, White_and_Purple, White_and_Yellow, Yellow_and_Purple. The icon sprites should be in the center rather than the upper left corner.
- Some of these items are redundant, most notably Captain_Blue_and_Gold and Captain_Gold_and_Blue. You should just choose one of those sprites and have it just be one item. Some of the other maintainers might notice other redundant items but that's just the main one that stands out to me.
- You should probably have the job-related socks not in the pride vend but instead in each department's respective vendor and/or locker. The captain's socks especially should be moved to the captain's locker. I also say this so that the pride vend doesn't just become the sock vend, as tempting as that may be to do.
- A lot of the item descriptions could use some work. I can go into more detail when I have more brain power, but for example one that sticks out is the item description of the N Parka, which has a gramatical mistake (it should say "A white parka with a purple N. It smells of pudding")

Other maintainers may have other things they want you to fix but that's just the stuff that stuck out to me at first glance. I'm always supportive of more customization options, so I would like to see this put in the game after a bit of polish.

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml:
```diff
@@ -17,4 +17,25 @@
+       Socks_Striped_White_and_Purple: 2
+       Socks_Striped_White_and_Red: 2
+       Socks_Striped_White_and_Yellow: 2
+       Socks_Striped_White_and_Purple: 2
```
This is mentioned twice, which is causing an error in the build.

Neptune-Zvezda replied:
i am a little confused on what part of this is mentioned twice or do you mean just in the error log and have no clue why ?

Neptune-Zvezda replied:
went to fix the LoQm issue and noticed i failed to save over the pride vendor inventory to remove the Department socks so clearly i need to double check more of my work

DangerRevolution commented:
rsi validator is saying that multiple of your sprites aren't 32x32 sized

DangerRevolution commented on Resources/Textures/DeltaV/Clothing/Under/Socks/Socks_Striped_Security.rsi/meta.json:
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039",
```
Up to you but I personally always said "resprited by DangerRevolution from TGstation..." just to watermark it.

DangerRevolution replied:
I would actually strongly recommend this; not only for you to follow the actual accreditation rules but to also have your name on it to credit the hours you spent spriting this.

Neptune-Zvezda replied:
Okay Modified Json's to properly include the suggested version of WaterMark

DangerRevolution commented on Resources/Textures/DeltaV/Clothing/Under/Socks/Socks_Striped_White_and_Orange.rsi/Socks_Striped_White_and_Orange.yml:
```diff
@@ -0,0 +1,15 @@
+	# Created From Coder Socks as Base for texturing
```
Random space

DangerRevolution commented on Resources/Textures/DeltaV/Clothing/Under/Socks/Socks_Striped_Cargo_LO-Qm.rsi/meta.json:
```diff
@@ -0,0 +1,18 @@
+{
```
One of the errors is about invalid file names; I'm not sure what symbols are allowed but it's worth checking if changing the - to a _ is better?

DangerRevolution commented:
In coder.rsi, it might be just how it's formatted for me on mobile but it looks like there's a space between the r and the .?

DangerRevolution commented:
> - Some of the icon sprites aren't centered properly. These would be Captain_Blue_and_Gold, Captain_Blue_and_White, Captain_Gold_and_Blue, Cargo, Cargo_LO-QM, Atmos, Engineering, Science, Security, White_and_Purple, White_and_Yellow, Yellow_and_Purple. The icon sprites should be in the center rather than the upper left corner.

> - Some of these items are redundant, most notably Captain_Blue_and_Gold and Captain_Gold_and_Blue. You should just choose one of those sprites and have it just be one item. Some of the other maintainers might notice other redundant items but that's just the main one that stands out to me.

> - You should probably have the job-related socks not in the pride vend but instead in each department's respective vendor and/or locker. The captain's socks especially should be moved to the captain's locker. I also say this so that the pride vend doesn't just become the sock vend, as tempting as that may be to do.

I would +1 to all these points, particularly point 3 with each locker recieving their appropriate sock options as that'd be interesting for outfits! + not overwhelming pride-o-mat!

IamVelcroboy commented:
Add a chance for these to show up in dressers `Entities/Structures/Furniture/dresser.yml`

IamVelcroboy commented:
Oh...and washing machines `Prototypes/Nyanotrasen/Entities/Structures/Machines/laundry.yml`

Colin-Tel commented on Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml:
```diff
@@ -17,4 +17,25 @@
+       Socks_Striped_White_and_Purple: 2
+       Socks_Striped_White_and_Red: 2
+       Socks_Striped_White_and_Yellow: 2
+       Socks_Striped_White_and_Purple: 2
```
```suggestion
       Socks_Striped_White_and_Purple: 2
       Socks_Striped_White_and_Red: 2
       Socks_Striped_White_and_Yellow: 2
```
This entry is listed twice.

Colin-Tel commented on Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml:
```diff
@@ -17,4 +17,25 @@
+       Socks_Striped_White_and_Yellow: 2
+       Socks_Striped_White_and_Purple: 2
+       Socks_Striped_Yellow_and_Purple: 2
+       N_Parka 2
```
```suggestion
       N_Parka: 2
```
Missing colon

DebugOk reviewed (changes requested):
Item names do not follow the requirements, do not merge without me having re-reviewed this

Neptune-Zvezda committed:
Sock bupdate .3

Moved Items to Delta V Directory as well as condensing into singular Yml files

Neptune-Zvezda committed:
Sock update .4

removed all _ from names of files and reading

Neptune-Zvezda commented:
> > * Some of the icon sprites aren't centered properly. These would be Captain_Blue_and_Gold, Captain_Blue_and_White, Captain_Gold_and_Blue, Cargo, Cargo_LO-QM, Atmos, Engineering, Science, Security, White_and_Purple, White_and_Yellow, Yellow_and_Purple. The icon sprites should be in the center rather than the upper left corner.
> 
> > * Some of these items are redundant, most notably Captain_Blue_and_Gold and Captain_Gold_and_Blue. You should just choose one of those sprites and have it just be one item. Some of the other maintainers might notice other redundant items but that's just the main one that stands out to me.
> 
> > * You should probably have the job-related socks not in the pride vend but instead in each department's respective vendor and/or locker. The captain's socks especially should be moved to the captain's locker. I also say this so that the pride vend doesn't just become the sock vend, as tempting as that may be to do.
> 
> I would +1 to all these points, particularly point 3 with each locker recieving their appropriate sock options as that'd be interesting for outfits! + not overwhelming pride-o-mat!

The Questionable Naming is more due to the fact they are different patterns which although fairly minor change how the sock may look on the person, as the Blue first merges a bit with the standard coat which gives a almost shorts feel which some may not like. The team can choose to change it to just one if felt like but i would rather the option firsthand.

Neptune-Zvezda commented:
> I like the idea of this, and most of the spritework is good. I have a couple of nitpicks though.
> 
> * Some of the icon sprites aren't centered properly. These would be Captain_Blue_and_Gold, Captain_Blue_and_White, Captain_Gold_and_Blue, Cargo, Cargo_LO-QM, Atmos, Engineering, Science, Security, White_and_Purple, White_and_Yellow, Yellow_and_Purple. The icon sprites should be in the center rather than the upper left corner.
> * Some of these items are redundant, most notably Captain_Blue_and_Gold and Captain_Gold_and_Blue. You should just choose one of those sprites and have it just be one item. Some of the other maintainers might notice other redundant items but that's just the main one that stands out to me.
> * You should probably have the job-related socks not in the pride vend but instead in each department's respective vendor and/or locker. The captain's socks especially should be moved to the captain's locker. I also say this so that the pride vend doesn't just become the sock vend, as tempting as that may be to do.
> * A lot of the item descriptions could use some work. I can go into more detail when I have more brain power, but for example one that sticks out is the item description of the N Parka, which has a gramatical mistake (it should say "A white parka with a purple N. It smells of pudding")
> 
> Other maintainers may have other things they want you to fix but that's just the stuff that stuck out to me at first glance. I'm always supportive of more customization options, so I would like to see this put in the game after a bit of polish.

addionally have now updated the Job based Socks to be inside the Drobe vendors for the respective jobs, addionally thanks for correcting the N parka (personal outfit), i do agree the decriptions can use work and would rather have one of the other devs suggest / tell what i could put best for them without just a overly basic "Socks that are ____ colors"
and the Sprites were uploaded wrong due to a weird software quirk with how i exported the Project file to my batch exporter and will be fixing it once they are online

Neptune-Zvezda commented:
> > * Some of the icon sprites aren't centered properly. These would be Captain_Blue_and_Gold, Captain_Blue_and_White, Captain_Gold_and_Blue, Cargo, Cargo_LO-QM, Atmos, Engineering, Science, Security, White_and_Purple, White_and_Yellow, Yellow_and_Purple. The icon sprites should be in the center rather than the upper left corner.
> 
> > * Some of these items are redundant, most notably Captain_Blue_and_Gold and Captain_Gold_and_Blue. You should just choose one of those sprites and have it just be one item. Some of the other maintainers might notice other redundant items but that's just the main one that stands out to me.
> 
> > * You should probably have the job-related socks not in the pride vend but instead in each department's respective vendor and/or locker. The captain's socks especially should be moved to the captain's locker. I also say this so that the pride vend doesn't just become the sock vend, as tempting as that may be to do.
> 
> I would +1 to all these points, particularly point 3 with each locker recieving their appropriate sock options as that'd be interesting for outfits! + not overwhelming pride-o-mat!

Neptune-Zvezda commented:
Frick wrong buttons

Neptune-Zvezda committed:
Outfit Update .5

Moved all Department Themed Socks to respective __-drobes Certain socks to Department Head Lockers
Last todo's Fix Image Sizes ( to be done)
add standard socks to certain random spawn pools

Neptune-Zvezda committed:
Outfit update .6 The Broke Server Bupdate

For real fixed the icons this time however test enviroment server was acting extremely unstable with ajustment of the Yml's from \ Backslashes to / forward slashes to match existing file formating

DangerRevolution commented on Resources/Textures/DeltaV/Clothing/Under/Socks/SocksStripedCaptainBlueandGold.rsi/meta.json:
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by Neptunezvezda(discord) Modified from "Coder Socks" tgstation https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039",
```
I have no clue about .json's rules but I'm not sure you can insert " speech marks in the middle of a different speech; use the ' apostrophe marks instead.

DangerRevolution commented on Resources/Textures/DeltaV/Clothing/Under/Socks/SocksStripedSecurityJackboots.rsi/meta.json:
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": 'Made by Neptunezvezda(discord) Modified from "Coder Socks" tgstation https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039",
```
Start of this line has an apostrophe, ', instead of speech and Coder Socks should be surrounded by ' marks instead of speech marks.

DangerRevolution commented on Resources/Textures/DeltaV/Clothing/Under/Socks/SocksStripedWhiteandGreen.rsi/meta.json:
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": Made by Neptunezvezda(discord) Modified from "Coder Socks" tgstation https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039",
```
This entirely lacks a speech mark at the start.

Neptune-Zvezda commented:
"server Log showcasing the error that occurs on attempts to start the round
[message.txt](https://github.com/DeltaV-Station/Delta-v/files/13753015/message.txt)
( i tried to link it within the main submition and it just put the full text wtf?

DangerRevolution commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSocks.yml:
```diff
@@ -0,0 +1,205 @@
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripedEngieeringAtmos
```
"Engieering"

Neptune-Zvezda committed:
Outfit update .7

Removed " " from Json's around coder socks as could have been causing issues as well as syncing inventory with master due to new winter boot addions

Neptune-Zvezda committed:
outfit update .72

Adjusted Json to follow Requested Copyright More Closely

Neptune-Zvezda committed:
Outfit Update .7.3

Refixed Pride Vendor stock as seemed to have forgotten to save changes in the past,
Renamed the Logi Socks to not include a - to avoid errors
Renamed Atmos Socks to have correct spelling of engineering on Files and yml
Remembered to Add the Qm's Socks to Locker

Neptune-Zvezda committed:
Merge branch 'master' into Neptune-Outfit-Submitions

Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
outfit update .7.4

i forgot to add DeltaV to parka sprite code

Neptune-Zvezda committed:
Merge branch 'Neptune-Outfit-Submitions' of https://github.com/Neptune-Zvezda/Delta-v-Nep-fuckery into Neptune-Outfit-Submitions

Colin-Tel commented on Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:
```diff
@@ -143,6 +146,7 @@
       - id: ClothingShoesBootsLaceup # DeltaV - add fancy shoes for HoP and cap
       - id: ClothingShoesMiscWhite # DeltaV - add fancy shoes for HoP and cap
       - id: ClothingShoesBootsWinterHeadOfPersonel #Delta V: Add departmental winter boots
+      - id: SocksStripedCaptainBlueandWhite
```
```suggestion
      - id: SocksStripedCaptainBlueandWhite # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:
```diff
@@ -102,6 +102,9 @@
       - id: ClothingShoesBootsWinterCap #Delta V: Add departmental winter boots
+      - id: SocksStripedCaptainBlueandGold
+      - id: SocksStripedCaptainBlueandWhite
+      - id: SocksStripedCaptainGoldandBlue
```
```suggestion
      - id: SocksStripedCaptainBlueandGold # DeltaV - Neptune's striped socks
      - id: SocksStripedCaptainBlueandWhite # DeltaV - Neptune's striped socks
      - id: SocksStripedCaptainGoldandBlue # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:
```diff
@@ -199,7 +203,7 @@
       - id: CEIDCard # Delta-V
       - id: ClothingShoesBootsWinterChiefEngineer #Delta V: Add departmental winter boots
-
+      - id: SocksStripedEngineering
```
```suggestion
      - id: SocksStripedEngineering # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:
```diff
@@ -254,7 +258,7 @@
       - id: CMOIDCard # Delta-V
       - id: ClothingShoesBootsWinterChiefMedicalOfficer #Delta V: Add departmental winter boots
-
+      - id: SocksStripedWhiteandLightBlueBunnySlipper
```
```suggestion
      - id: SocksStripedWhiteandLightBlueBunnySlipper # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:
```diff
@@ -374,6 +378,7 @@
       - id: WeaponEnergyGunMultiphase # DeltaV - HoS Energy Gun
       - id: HoSIDCard # Delta-V
       - id: ClothingShoesBootsWinterHeadOfSecurity #Delta V: Add departmental winter boots
+      - id: SocksStripedSecurityJackboots
```
```suggestion
      - id: SocksStripedSecurityJackboots # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/atmosdrobe.yml:
```diff
@@ -14,3 +14,4 @@
     ClothingOuterWinterAtmos: 2
     ClothingNeckScarfStripedLightBlue: 3
     ClothingShoesBootsWinterAtmos: 2 #Delta V: Add departmental winter boots
+    SocksStripedEngineeringAtmos: 3
```
```suggestion
    SocksStripedEngineeringAtmos: 3 # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml:
```diff
@@ -15,3 +15,5 @@
     ClothingNeckScarfStripedBrown: 3
     ClothingShoesBootsWinterCargo: 2
+    SocksStripedCargo: 3
+    SocksStripedCargoLO-Qm: 3
```
```suggestion
    SocksStripedCargo: 3 # DeltaV - Neptune's striped socks
    SocksStripedCargoLO-Qm: 3 # DeltaV - Neptune's striped socks

```

Colin-Tel commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/chemdrobe.yml:
```diff
@@ -14,3 +14,4 @@
     ClothingHeadsetMedical: 2
     ClothingOuterWinterChem: 2
     ClothingShoesBootsWinterChem: 2 #Delta V: Add departmental winter boots
+    SocksStripedWhiteandOrange: 2
```
```suggestion
    SocksStripedWhiteandOrange: 2 # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/engidrobe.yml:
```diff
@@ -16,3 +16,4 @@
     ClothingOuterWinterEngi: 2
     ClothingNeckScarfStripedOrange: 3
     ClothingShoesBootsWinterEngi: 2
+    SocksStripedEngineering: 5
```
```suggestion
    SocksStripedEngineering: 5 # DeltaV - Neptune's striped socks

```

Colin-Tel commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/medidrobe.yml:
```diff
@@ -17,7 +17,10 @@
     UniformScrubsColorPurple: 4
     ClothingShoesBootsWinterMed: 2
+    SocksStripedWhiteandLightBlueBunnySlipper: 3
+    SocksStripedWhiteandLightBlue: 3
```
```suggestion
    SocksStripedWhiteandLightBlueBunnySlipper: 3 # DeltaV - Neptune's striped socks
    SocksStripedWhiteandLightBlue: 3 # DeltaV - Neptune's striped socks

```

Colin-Tel commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/scidrobe.yml:
```diff
@@ -16,3 +16,4 @@
     ClothingNeckScarfStripedPurple: 3
     ClothingHeadTinfoil: 2 # Nyanotrasen - Tinfoil hats for Epistemics
     ClothingShoesBootsWinterSci: 2
+    SocksStripedScience: 3
```
```suggestion
    SocksStripedScience: 3 # DeltaV - Neptune's striped socks
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSocks.yml:
```diff
@@ -0,0 +1,205 @@
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStripped WhiteandLightBlue
```
```suggestion
  id: SocksStripedWhiteandLightBlue
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSocks.yml:
```diff
@@ -0,0 +1,205 @@
+  description: Striped Socks that make you feel special enough to boss the heads
+  components:
+  - type: Sprite
+    sprite:DeltaV/Clothing/Under/Socks/SocksStripedCaptainGoldandBlue.rsi
```
```suggestion
sprite: DeltaV/Clothing/Under/Socks/SocksStripedCaptainGoldandBlue.rsi
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSocks.yml:
```diff
@@ -0,0 +1,205 @@
+
+- type: entity
+  parent: ClothingShoesBase
+  id: SocksStrippedCargo
```
```suggestion
  id: SocksStripedCargo
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSocks.yml:
```diff
@@ -0,0 +1,205 @@
+  parent: ClothingShoesSlippers
+  id: SocksStripedWhiteandLightBlueBunnySlipper
+  name: Striped Socks with Bunny Slippers
+  description: a Pair of Bunny Slipers with striped socks for all your hospital needs.
```
```suggestion
  description: A pair of bunny slippers with striped socks for all your hospital needs.
```

Colin-Tel reviewed (changes requested):
All prototype IDs need to change to fit the standard clothing format, all item names need to change to be more uniform, the additions in heads' filled lockers need to exist in their respective hardsuit lockers as well.

Neptune-Zvezda committed:
Apply suggestions from code review

Thank you colin!

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSocks.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
update test 7.4.1.1.1.1.1.1.1.1.1

Neptune-Zvezda committed:
Merge branch 'Neptune-Outfit-Submitions' of https://github.com/Neptune-Zvezda/Delta-v-Nep-fuckery into Neptune-Outfit-Submitions

Neptune-Zvezda committed:
outfit update 7.4.2.2.2.2.2.2.2.2.2

im fixing the Names of "Equip-FEET" to not be capitialized

Colin-Tel committed:
Update StripedSocks.yml

:trollface:

Neptune-Zvezda committed:
I have killed socks

death

Neptune-Zvezda committed:
i have unkillted socks

undeath

Neptune-Zvezda committed:
suffering

Neptune-Zvezda committed:
LOOOOO

Neptune-Zvezda committed:
update .7.5

Colin-Tel committed:
Update cargo_vending.yml

adjust price

Neptune-Zvezda committed:
Updated N parka

Updated N parka to move sleeves one pixel in as well as fixing a stray pixel on the front view

Neptune-Zvezda committed:
Outfit Update .8

Adjusted all Item Id to include ClothingUnder as to standardise for researching,
Socks Should now be eatable by moffs
N parka now has storage

Neptune-Zvezda committed:
Merge branch 'master' into Neptune-Outfit-Submitions

Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Outfit update .8.1

Fixed ID Name in Medidrobe as was not updated with ClothingUnder like other drobes

Neptune-Zvezda committed:
Merge branch 'master' into Neptune-Outfit-Submitions

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/Cargo/cargo_vending.yml:
```diff
@@ -4,6 +4,6 @@
     state: base
   product: CrateVendingMachineRestockPrideFilled
-  cost: 1000
+  cost: 1250
```
This should be mentioned in your Changelog too.

Neptune-Zvezda replied:
i Believe i have added this to the change log now but correct me if i did it poorly

Neptune-Zvezda committed:
Outfit Update .8.2

Patched N parka and atmos socks to have correct Id in Yml's to match other items

Neptune-Zvezda committed:
Outfit Update .8.3

Updated N parka Names to not Include _'s to standardize naming

Neptune-Zvezda committed:
outfit update .8.4

Changed Sock .Yml EngineeringAtmos to be / and not \  (got to read update with rename of folder)

Neptune-Zvezda committed:
Outfit Update .8.5

Finally Noticed Socks was in the Id Name for Atmos Socks Twice

Neptune-Zvezda committed:
Outfit update .9.0

Captain's locker was too filed to fit the socks, moved to pride vendor until futher ideas

DangerRevolution commented on Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml:
```diff
@@ -17,4 +17,16 @@
+       ClothingUnderSocksStripedYellowandPurple: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedCaptainBlueandGold # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedCaptainBlueandWhite # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedCaptainGoldandBlue # DeltaV - Neptune's striped socks 
```
Line 29, 30 and 31 are all missing correct formatting, which is what's causing your error

Neptune-Zvezda committed:
outfit update .9.1

Socks Moved From captain locker were not given a amount to be stocked with.

IamVelcroboy commented on Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:
```diff
@@ -66,6 +66,7 @@
       - id: ClothingShoesMiscWhite # DeltaV - add fancy shoes for HoP and cap
       - id: ClothingShoesBootsWinterCap #Delta V: Add departmental winter boots
 
```
```yml
    - id: ClothingUnderSocksStripedCaptainBlueandGold 
      prob: 0.3
      orGroup: socks 
    - id: ClothingUnderSocksStripedCaptainBlueandWhite 
      prob: 0.3
      orGroup: socks 
    - id: ClothingUnderSocksStripedCaptainBlueandWhite 
      prob: 0.3
      orGroup: socks 
```

Neptune-Zvezda committed:
Outfit update .9.2

Adjusted Cost of pride restock as with addion of "captain" socks went over the cost limit

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedWhiteandLightBlue
+  name: light blue striped socks
+  description: Wearing these makes you feel just a little bit lazier
```
```suggestion
  name: striped socks
  description: Wearing these white and light blue socks makes you feel just a little bit lazier.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedCaptainBlueandGold
+  name: Blue and gold striped socks
+  description: Striped Socks that make you feel special enough to boss the heads
```
```suggestion
  name: striped socks
  description: Striped socks in blue and gold that make you feel special enough to boss the heads.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedCaptainBlueandWhite
+  name: Blue striped socks
+  description: Striped Socks that make you feel special enough to boss the heads
```
```suggestion
  name: striped socks
  description: Striped socks in blue and white that make you feel special enough to boss the heads.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedCaptainGoldandBlue
+  name: Gold and blue striped socks
+  description: Striped Socks that make you feel special enough to boss the heads
```
```suggestion
  name: striped socks
  description: Striped socks in gold and blue that make you feel special enough to boss the heads.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedCargo
+  name: Cargo striped socks
+  description: These Striped Socks make you feel a little bit stronger
```
```suggestion
  name: striped socks
  description: These striped socks make you feel a little bit stronger.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedCargoQm
+  name: Logistics striped socks
+  description: Striped Socks that make you feel like becoming an independant Station State
```
```suggestion
  name: striped socks
  description: Striped socks that make you feel like becoming an independent state.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedEngineeringAtmos
+  name: Atmos Tech striped socks
+  description: Striped socks to make you stand out in an emergency
```
```suggestion
  name: striped socks
  description: Striped socks to make you stand out in an atmospheric emergency.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedEngineering
+  name: Engineer striped socks
+  description: Striped socks to make you stand out in an emergency
```
```suggestion
  name: striped socks
  description: Striped socks to make you stand out in an emergency.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedScience
+  name: Science striped socks
+  description: These Striped socks make you want to do some Science
```
```suggestion
  name: striped socks
  description: These striped socks make you want to do some science.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedSecurity
+  name: Security striped socks
+  description: These striped socks make you feel like beating up a mime
```
```suggestion
  name: striped socks
  description: These striped socks make you feel like beating up an evildoer.
```
Trying not to allude to brutality being okay.

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesMilitaryBase
+  id: ClothingUnderSocksStripedSecurityJackboots
+  name: JackBoots with striped socks
+  description: The striped socks make you feel a little more "Secure"
```
```suggestion
  name: jackboots with striped socks
  description: The striped socks make you feel a little more "secure."
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedWhiteandGreen
+  name: Green striped socks
+  description: Minty! Now in Green.
```
```suggestion
  name: striped socks
  description: Minty! Now in Green.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+- type: entity
+  parent: ClothingShoesSlippers
+  id: ClothingUnderSocksStripedWhiteandLightBlueBunnySlipper
+  name: Striped socks with bunny slippers
+  description: A pair of bunny slippers with striped socks for all your hospital needs.
```
```suggestion
  name: slippers with striped socks
  description: A pair of bunny slippers with striped socks for all your hospital needs.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedWhiteandOrange
+  name: Orange striped socks
+  description: Striped Socks that match the traffic style
```
```suggestion
  name: striped socks
  description: Striped socks that match the traffic style.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedWhiteandPurple
+  name: Purple striped socks
+  description: Striped Socks that smell like Grape Soda
```
```suggestion
  name: striped socks
  description: Striped socks that smell like grape soda.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedWhiteandRed
+  name: Red striped socks
+  description: Minty! Now in red
```
```suggestion
  name: striped socks
  description: Minty! Now in red.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedWhiteandYellow
+  name: yellow striped socks
+  description: Please don't order pilk at the bar
```
```suggestion
  name: striped socks
  description: Please don't order pilk at the bar.
```

Colin-Tel commented on Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml:
```diff
@@ -0,0 +1,438 @@
+  parent: ClothingShoesBase
+  id: ClothingUnderSocksStripedYellowandPurple
+  name: Yellow and purple striped socks
+  description: Striped Socks styled after banana and grape jelly sandwiches
```
```suggestion
  name: striped socks
  description: Striped socks styled after banana and grape jelly sandwiches.
```

Colin-Tel reviewed (commented):
Take a look over these and consider them. If they're alright with you, then please do these changes in one batch or I'll cry.

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update Resources/Prototypes/DeltaV/Entities/Clothing/Under/StripedSock.yml

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Neptune <154393072+Neptune-Zvezda@users.noreply.github.com>

Neptune-Zvezda committed:
Update heads.yml

Neptune-Zvezda committed:
Merge branch 'Neptune-Outfit-Submitions' of https://github.com/Neptune-Zvezda/Delta-v-Nep-fuckery into Neptune-Outfit-Submitions
